src/main/resources/hudson/tasks/junit/JUnitResultArchiver/config.jelly

```diff
@@ -33,6 +33,9 @@ THE SOFTWARE.
 	<f:entry field="keepLongStdio" title="">
 		<f:checkbox name="keepLongStdio" title="${%Retain long standard output/error}"/>
 	</f:entry>
+	<f:entry title="${%Suite name prefix}" field="suitePrefix">
```

Comment:
I have no strong opinion on this topic< but for me this option seems to be too narrow specific to be kept in the main config